### DISCUSSION TITLE
Drop PouchDB-Server from CI?

### DISCUSSION THREAD

**Discussion by SourceR85** [pouchdb/pouchdb#8872] [Ideas]
Hi,
I got a big question for you: should we keep pouchdb-server as a build dependency?

The reason for this question: pouchdb-server hasn't seen an update in half a decade and for me, it just seems like it's another failing point (unmaintained, incompatible with CouchDB 3.x, outdated dependencies, CJS only).

If you agree, I'll write a PR to drop pouchdb-server from PouchDB's CI.

**Comment by garethbowen** [comment]
Yeah I think so. I think it's still useful for things like testing so you don't have to spin up a whole CouchDB server but it's obviously not being maintained so there's no point blocking CI on it.

**Reply by SourceR85** [reply]
Would you share a common use case, where a pouchdb-server is beneficial?

From resource point of view, I don't see big differences:
If I spin up couchdb:3.1 in docker, it uses less than 0.5% CPU time and ~70MB RAM.
Starting pouchdb-server seems roughly the same (60-70MB RAM, ~0.2% CPU time).
(My setups: Ryzen 7 6800HS in a laptop and Ryzen 7 3800X in a PC... In both cases, the performance draw isn't noticeable.)

**Reply by garethbowen** [reply]
Was that using the in-memory option? I would expect it to be much faster. Using it in memory also means it's cleaned up easily which is really useful for integration tests.

But as I said, I agree with removing it from CI.

**Reply by SourceR85** [reply]
I was just interested, how someone can benefit from it and if it's worth for me to invest time in updating pouchdb-server 😉

**Comment by SourceR85** [comment]
Will be closed with merge of #8948.
